NSS-Day-Cohort-49/giffygram-double-rainbows
# Added UserProfile.js and the userProfile module

#### Changes Made
1. Added file `UserProfile.js` to `feed` directory.
1. Modified file `UserProfile.js` to include `userProfile` that filters and displays of the post of the selected user.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout user-profile-page
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
